### PR TITLE
tests: Remove redundant verify

### PR DESCRIPTION
**Pull request by kingiler**
Address #1075

### CODE REVIEW

File: tests/test_attribute_definition.py [hunk 1]
```diff
@@ -3,24 +3,25 @@
...
+    ParamAttrDef,
     ParameterDef,
     irdl_attr_definition,
     irdl_to_attr_constraint,
-    AnyAttr,
-    BaseAttr,
-    ParamAttrDef,
 )
 from xdsl.parser import AttrParser
 from xdsl.printer import Printer
```
Comment (webmiche) [inline tests/test_attribute_definition.py:27]: These changes seem unrelated. Can you remove them? (check out `git add -p` to add parts of files)

Comment (kingiler) [reply tests/test_attribute_definition.py]: Addressed in #1243.

**Review by math-fehr — COMMENTED** [review]
Thanks!
